docs/analog/commands/GetArchiveHoursMCEx.md

```diff
@@ -0,0 +1,130 @@
+# GetArchiveHoursMCEx
+
+Command to request/receive hour pulse counter's values from device archive. This is an extended version of [GetArchiveHoursMC](./GetArchiveHoursMC.md) to get archive hours more than 8
+
+## Request
+
+### Format
+
+| Size   | Type                                         | Field                                            |
+| ------ | -------------------------------------------- | ------------------------------------------------ |
+| `1`    | `byte`                                       | extra flag = `0x1f`                              |
```

Comment:
```suggestion
| `1`    | `uint8`                                       | extra flag = `0x1f`                              |
```

type `byte` not exists anymore, use: int8, int16, uint8, etc.

like here - https://github.com/jooby-dev/jooby-docs/blob/main/docs/analog/commands/GetBatteryStatus.md#format-1